lib/ex_doc/formatter/html/templates/head_template.eex

```diff
@@ -11,6 +11,9 @@
       <link rel="canonical" href="<%= config.canonical %>" />
     <% end %>
     <script src="<%= asset_rev config.output, "dist/sidebar_items*.js" %>"></script>
+    <%= if config.javascript_script_path do %>
+      <script src="<%= config.javascript_script_path %>"></script>
```

Comment:
What do you think about `javascript_config_path` to indicate this is a path to configure what is happening on the javascript side of things? :D